lib/testing-helpcenter/help_center_tags.rb

```diff
@@ -1,16 +1,24 @@
 # frozen_string_literal: true
 
+require_relative 'articles_with_tag_popup'
+
 # Methods to work with tag list on https://helpcenter.onlyoffice.com/tags.aspx
 class HelpCenterTags
   include PageObject
 
   def initialize(instance)
     super(instance.webdriver.driver)
     @instance = instance
-    @xpath = "//div[contains(@class, 'div_for_index')]//a[contains(@class, 'text-container-link')]"
+    @tags_xpath = "//div[contains(@class, 'div_for_index')]//a[contains(@class, 'text-container-link')]"
   end
 
   def tags_list
-    @instance.webdriver.get_text_of_several_elements(@xpath)
+    @instance.webdriver.get_text_of_several_elements(@tags_xpath)
+  end
+
+  def tag_click(tag_name, tags_array)
+    tag_index = (tags_array.find_index(tag_name) + 1).to_s
+    @instance.webdriver.get_element("//div[contains(@class, 'div_for_index')]//li[#{tag_index}]//a[contains(@class, 'text-container-link')]").click
```

Comment:
I think this xpath can be simplified in something like `(#{tags_xpath})[#{tag_index + 1}]`